src/test/java/br/com/uniamerica/gajigo/unit/repository/UserRepositoryTest.java

```diff
@@ -31,4 +39,26 @@ public void testFindUsername() {
 
         assertEquals(a, findA);
     }
+
+    @Test
+    public void testInsertUser() {
+        User user = new User("eduardo123", "minhasenha123", "eduardo de souza magalhaes");
+        user.setEmail("eduardo@gmail.com");
+        this.repository.save(user);
```

Comment:
Na maioria do código pulamos o uso do `this`. Aqui também não é necessario usar.

Reply:
Review não resolvido ainda. As mudanças que falo nas reviews se aplicam para todos os arquivos que foram mudados, não só no trecho que o github da highlight, até porque pelo que eu sei o github não deixa dar highlight em mais do que um trecho por review.

Reply:
@pehdfms foi decidido, em conjunto, que será usado o this